### PR TITLE
Economy: Phase 1

### PR DESCRIPTION
**Pull request by NanakoAC**
The first of many upcoming economy PRs
This PR includes three main changes to guild financing.

1. Supply points removed.
The entire concept of supply points is gone. They were originally designed to represent internal requisition within a large corporation (Nanotrasen) and are not suitable for our much more Ancap universe.
The guild now purchases things using money, directly from the guild department account.  There is no passive tick of funding, the guild account will not magically recieve money for doing nothing. It makes money by selling things to players.
The merchant is generally expected to make people pay for the things they order. It should be possible to use existing devices like EFTPOS to enable card payments directly to the guild account, but accepting cash and manually depositing it is also always an option.

In future, i plan to add a cash register and more ATMs in cargo to make this smoother
As a nice little bonus, the transaction information includes what was purchased, and by who. The merchant can poke through the transaction history of the guild account when he's bored, and see exactly who has purchased what, across the entire shift

2. The Guild operates Vendors
Now, all vendors are operated by the guild. Whenever anyone buys anything from a vendor, their credits are transferred directly to the guild's department account, and thus can be used for orders.
In future, we will probably add the ability for vendors to be registered to different departments, and thus allow a department to sell its supplies that way.

3. Centcom Selling Removed
This refers to the feature whereby players load items onto the supply shuttle and send it away. Everything on the shuttle is deleted, and some supply points are recieved for certain things on it (crates and materials).
There are several problems with this feature which justify its removal

-------------------------------------------------------
1. Destruction of content. Everything on it is deleted, and most of that doesnt even have value. It just turns a bunch of items to ash for no purpose, its silly to give players an easy way to do that. More significantly, i've observed many players doing this just because they didn't care for the things they had. They ship it off on the shuttle because they don't see any current use for it. This is bad

2. Discourages interaction. Most of the things that get shipped out like this are the starting content that spawns in the cargo bay at roundstart. These items exist to provide some basic stock for the guild to sell to players, and in general, the guild's purpose is to sell items to other players, and thus promote interaction between players. Selling everything to a faceless nonentity defeats this purpose.

3. There is no centcomm. We're a ship in deep space, not a nanotrasen station. Lorewise, there isn't anyone to sell this stuff to. Technically there's nobody to order it from either. In the future we will be implementing NPC traders who buy and sell limited ranges of items, and we can repurpose this functionality then in order to facilitate that

4. No longer necessary. With the removal of supply points, and many ways to get money by interacting with other players, there's no longer any mechanical need to manufacture points by selling crates to centcomm
-------------------------------------
In short, this feature encourages silly/undesireable behaviour in players, and works against design goals. It doesnt need to exist, so i've removed it. Now if you send anything off on the shuttle, it will simply be sitting there where you left it when the shuttle comes back. The shuttle is only for recieving things, for now.

Other small changes:
Renamed a significant number of instances across the codebase where the guild department was still called Cargo.

The supply ordering console no longer shows guild finances. This is the publicly accessible one where people place orders. The ones inside the guild which are guild access only, will accurately display the contents of the guild account

The Merchant's starting bank balance is removed. He no longer gets a specific salary, but instead is free to pay himself as much as he wants from the guild department account

### CODE REVIEW

File: code/controllers/subsystems/supply.dm [hunk 1]
```diff
@@ -34,10 +28,9 @@ SUBSYSTEM_DEF(supply)
 	return ..()
 
 /datum/controller/subsystem/supply/fire()
-	points += points_per_process
```
[the remark this answers is not in the document]
Comment (NanakoAC) [reply code/controllers/subsystems/supply.dm]: Probably not, i wasnt sure how to disable it though

[the remark this answers is not in the document]
Comment (LethalGhost) [reply code/controllers/subsystems/supply.dm]: Remove that proc (`/datum/controller/subsystem/supply/fire()`) and set supply subsystem flags to
`	flags = SS_NO_FIRE`

[the remark this answers is not in the document]
Comment (NanakoAC) [reply code/controllers/subsystems/supply.dm]: done

File: code/modules/economy/cash_register.dm [hunk 2]
```diff
@@ -546,7 +546,7 @@
 	..()
 
 /obj/machinery/cash_register/cargo
-	account_to_connect = "Cargo"
+	account_to_connect = "Guild"
```
Comment (mattroks101) [inline code/modules/economy/cash_register.dm]: This is commented out? Are we planning on making use of this later?

Comment (NanakoAC) [reply code/modules/economy/cash_register.dm]: yes indeed we are
I didn't activate them in this PR, beyond scope for now, not sure if they'd be bugged. but i'll do it in another future one soonish

**Pull request merged**